recipes/aws-sdk-cpp/all/conanfile.py

```diff
@@ -0,0 +1,418 @@
+            "network-firewall",
+            "networkmanager",
+            "opsworks",
+            "opsworkscm",
+            "organizations",
+            "outposts",
+            "personalize",
+            "personalize-events",
+            "personalize-runtime",
+            "pi",
+            "pinpoint",
+            "pinpoint-email",
+            "polly",
+            "polly-sample",
+            "pricing",
+            "qldb",
+            "qldb-session",
+            "queues",
+            "quicksight",
+            "ram",
+            "rds",
+            "rds-data",
+            "redshift",
+            "redshift-data",
+            "rekognition",
+            "resource-groups",
+            "resourcegroupstaggingapi",
+            "robomaker",
+            "route53",
+            "route53domains",
+            "route53resolver",
+            "s3",
+            "s3-encryption",
+            "s3control",
+            "s3outposts",
+            "sagemaker",
+            "sagemaker-a2i-runtime",
+            "sagemaker-edge",
+            "sagemaker-featurestore-runtime",
+            "sagemaker-runtime",
+            "savingsplans",
+            "schemas",
+            "sdb",
+            "secretsmanager",
+            "securityhub",
+            "serverlessrepo",
+            "service-quotas",
+            "servicecatalog",
+            "servicecatalog-appregistry",
+            "servicediscovery",
+            "sesv2",
+            "shield",
+            "signer",
+            "sms",
+            "sms-voice",
+            "snowball",
+            "sns",
+            "sqs",
+            "ssm",
+            "sso",
+            "sso-admin",
+            "sso-oidc",
+            "states",
+            "storagegateway",
+            "sts",
+            "support",
+            "swf",
+            "synthetics",
+            "text-to-speech",
+            "textract",
+            "timestream-query",
+            "timestream-write",
+            "transcribe",
+            "transcribestreaming",
+            "transfer",
+            "translate",
+            "waf",
+            "waf-regional",
+            "wafv2",
+            "wellarchitected",
+            "workdocs",
+            "worklink",
+            "workmail",
+            "workmailmessageflow",
+            "workspaces",
+            "xray"
+           )
+    options = {
+            **{ x: [True, False] for x in sdks},
+            **{
+                "shared": [True, False],
+                "fPIC": [True, False],
+                "min_size": [True, False],
+                }
+            }
+    default_options = {key: False for key in options.keys()}
+    default_options["fPIC"] = True
+    default_options["access-management"] = True
+    default_options["identity-management"] = True
+    default_options["queues"] = True
+    default_options["transfer"] = True
+    default_options["s3-encryption"] = True
+    default_options["text-to-speech"] = True
+    internal_requirements = {
```

Comment:
like for `sdk`, please prefix this attribute with `_`

Reply:
done